src/containers/Bookings.js

```diff
@@ -1,19 +1,62 @@
-import React, { Component } from 'react';
-import Search from '../components/Search.js';
-// import Results from '../components/Results.js';
-// import FakeBookings from '../data/fakeBookings.json';
+import React, { Component } from "react";
+import Search from "../components/Search.js";
+import BookingsMessage from "../components/BookingsMessage.js";
+import Results from "../components/Results.js";
+import fakeBookings from "../data/fakeBookings.json";
+import Counter from "../components/Counter";
 
 export default class Bookings extends Component {
-  search = () => {
-    console.info('TO DO!');
+  state = {
+    count: 0,
+    count2: 0,
+    sum: 0,
+    input: 0,
+    bookings: fakeBookings
   };
-  
+
+  userInput = e => {
+    e.preventDefault();
+    this.setState({
+      input: parseInt(e.target.value, 10)
+    });
+  };
+
+  bookingsById = () => {
+    this.setState({
+      bookings: fakeBookings.filter(booking => {
+        return booking.id === this.state.input;
+      })
+    });
+  };
+
+  increase = () => {
+    this.setState({
+      count: ++this.state.count,
+      sum: ++this.state.sum
+    });
+  };
+
+  increase2 = () => {
+    this.setState({
+      count2: ++this.state.count2,
+      sum: ++this.state.sum
+    });
+  };
+
   render() {
     return (
       <div className="App-content">
         <div className="container">
-          <Search search={this.search} />
-          {/* <Results results={this.state.results} /> */}
+          <BookingsMessage />
+          <Search userInput={this.userInput} bookingsById={this.bookingsById} />
+          Results: {this.state.bookings.length}
+          <Results results={this.state.bookings} />
+          Sum: {this.state.sum}
+          <br />
+          {this.state.count}
+          <Counter name="Counter 1" increase={this.increase} />
```

Comment:
again .. I would name the prop `onIncrease` to make it obvious to people that it is an event.

Reply:
When you try to choose a meaningful variable name: 
https://instagram.com/p/BjcOSE8jo67/